modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/message/SnapshotTxDataRequest.java

```diff
@@ -25,4 +25,10 @@
  */
 @Transferable(TableMessageGroup.SNAPSHOT_TX_DATA_REQUEST)
 public interface SnapshotTxDataRequest extends SnapshotRequestMessage {
+    /**
+     * Returns maximum number of transactions that should be sent in response.
+     *
+     * @return Maximum number of transactions that should be sent in response.
```

Comment:
```suggestion
     * Returns maximum number of transactions that should be sent in response.
```

Reply:
IIRC, there is no strict rule imposing this. So I would prefer to leave it as is.